src/main/java/at/ac/tuwien/kr/alpha/solver/DefaultSolver.java

```diff
@@ -67,9 +72,11 @@
 	private boolean initialize = true;
 	private int mbtAtFixpoint;
 	private int conflictsAfterClosing;
+	private final boolean disableJustifications;
+	private boolean disableJustificationAfterClosing = true;	// Keep disabled for now, case not fully worked out yet.
```

Comment:
Please create an issue so we can track this.

Reply:
Done, issue #138.